src/coreclr/src/jit/instrsxarch.h

```diff
@@ -594,6 +594,7 @@ INST3(blsr,             "blsr",             IUM_WR, BAD_CODE,     BAD_CODE,
 INST3(bextr,            "bextr",            IUM_WR, BAD_CODE,     BAD_CODE,     SSE38(0xF7),                             INS_Flags_IsDstDstSrcAVXInstruction)    // Bit Field Extract
 
 // BMI2
+INST3(rorx,             "rorx",             IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0xF0),                             INS_FLAGS_None)
```

Comment:
Any plans for a similar change around `SARX`, `SHRX` and `SHLX`?

Reply:
Maybe in a separate PR? this is my first PR to add a new (missing) instruction. I tried to add sarx/shrx but faced a minor issue with "contained" for the arg (a redundant mov was emitted).